rest_framework/fields.py

```diff
@@ -778,6 +778,8 @@ def to_internal_value(self, data):
         return data
 
     def to_representation(self, value):
+        if value is None:
+            return None
```

Comment:
See the first commit here: https://github.com/carltongibson/django-rest-framework/commit/ab3f9cf6e0f5b30f2256a59526481159bb8205d4

(Before getting distracted with 1.7)

The test fails without this change. If this is the wrong fix, then where should it go? (I'm happy to dig but you probably Just Know™)